game/src/edit/traffic_signals/edits.rs

```diff
@@ -79,7 +114,9 @@ impl SimpleState<App> for ChangeDuration {
                 let new_type = if panel.is_checked("phase type") {
                     PhaseType::Fixed(dt)
                 } else {
-                    PhaseType::Adaptive(dt)
```

Comment:
Ah, this confused me a fair bit. I set the two new spinners but didn't toggle to adaptive, and I couldn't figure out why my changes weren't taking effect. How about we get rid of the `Checkbox::toggle` entirely and decide between `Fixed` and `Variable` based on if `delay` and `additional` are nonzero.

Reply:
I want to think about that. I agree the toggle is sub-optimal. I actually had wanted a radio button, to preserve Adaptive, but opted out of it. I've done that several times as well, forgetting to toggle. In my head I'm trying to reconcile if just having additional != 0 is sufficient. However, it leaves delay dangling out there with no meaning for a fixed signal, which I dislike almost as much as having two fields being used to indicate fixed. Originally, I considered disabling/enabling delay and additional based upon the checkbox.

Reply:
This is likely something that @muzlee1113 can help with. The API for doing it isn't very ergonomic, but you can make the two extra sliders appear/disappear based on something being checked. You'd listen for `Outcome::Changed`, then do `panel.replace()` to make things appear or disappear.

For the purposes of this PR, we can leave it as it is.